### PR TITLE
LPS-72641 Removed taglib_tags_selector groupIds filtering

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
It still needs the backend groupIds filtering in order to avoid unwanted exceptions

**Comment by p2kmgcl** [comment]
@jbalsas I think I have changed everything you mentioned. I have reported the backend error to @ealonso and, as I understood, it will be solved after merging these changes.

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.es.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ */
+class TagSelector extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	attached() {
+		AUI().use(
+			'liferay-asset-taglib-tags-selector',
+			function(A) {
+				this.input_ = this.element.querySelector('input[type="text"]');
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.es.js]: Hey @p2kmgcl, I just meant to move this declaration to the `attached` method, not everything.

Is this still working properly?

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.es.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	 * stablished inside attached() method.
+	 * @private
+	 */
+	focusTagInput_() {
+		this.input_.focus();
+	}
+}
+
+TagSelector.STATE = {
+	divId: Config.string().value(''),
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.es.js]: We're missing docs here 😉

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.soy [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{namespace TagSelector}
+
+/**
+ *
+ */
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.soy]: Missing docs 😉

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.soy [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+{template .render}
+	{@param index: int}
+	{@param? namespace: string}
+
+	<div id="{$namespace}assetTagsSelector{$index}">
+		<input name="{$namespace}queryTagNames{$index}" id="{$namespace}queryTagNames{$index}" type="hidden" />
+		<input class="form-control lfr-tag-selector-input" id="{$index}assetTagName" size="15" title="Add tags" type="text" />
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.soy]: Title should be localized

Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/TagSelector.soy]: This is done using the `{msg}add-tags{/msg}`

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/CategorySelector.es.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	 */
+	onEntriesChanged_() {
+		this.rule.categoryIdsTitles = this.categoriesSelector_.entries.values.map((element) => element.value);
+		this.rule.queryValues = this.categoriesSelector_.entries.keys.join(',');
+	}
+
+}
+
+CategorySelector.STATE = {
+	categorySelectorURL: Config.string().value(''),
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/CategorySelector.es.js]: Missing jsdocs

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/CategorySelector.es.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+/**
+ * CategorySelector is a temporary Component wrapping the existing
+ * AUI module liferay-asset-taglib-categories-selector
+ */
+class CategorySelector extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	rendered() {
+		AUI().use(
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/CategorySelector.es.js]: Should move this to `attached` if it works same way as the tags-selector

File: modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/AutoField.es.js [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+	 * - rules: [{}, {}, {}]
+	 * - queryLogicIndexes: "0,1,2";
+	 */
+	onRulesChanged_() {
+		this.queryLogicIndexes = Object.keys(this.rules).toString();
+	}
+}
+
+AutoField.STATE = {
+	/**
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/AutoField.es.js]: Missing jsdocs :)

**Comment by p2kmgcl** [comment]
#756

**Pull request closed without merging**